app/views/lessons/show.jbuilder

```diff
@@ -1,6 +1,7 @@
-json.post do
-  json.id @lesson.id
-  json.title @lesson.title
-  json.description @lesson.description
-  json.video_link @lesson.video_link
-end
+json.extract! @lesson, :id
+json.title @lesson.title
+json.description @lesson.description
+json.video_link @lesson.video_link
+json.author_id @lesson.author_id
+json.category_id @lesson.category_id
+json.status @lesson.status
```

Comment:
Better this? `json.extract! @lesson, :id, :title, :description, :video_link, :author_id, :category_id, :status`